src/main/fc/stats.c

```diff
@@ -87,19 +83,16 @@ void statsOnDisarm(void)
     int8_t minArmedTimeS = statsConfig()->stats_min_armed_time_s;
     if (minArmedTimeS >= 0) {
         uint32_t dtS = (millis() - arm_millis) / 1000;
+        statsConfigMutable()->stats_total_time_s += dtS;
```

Comment:
This seems to be wrong - if we are not counting a short arming timespan as a flight, then we should not be counting its duration as flight time.

Reply:
@mikeller maybe I am thinking about the use-case just for myself...
A flight counter is a subjective value. I am a racer and to me a flight is anything more than 30-60 secs. If I crash earlier I don't count that as a pack, but still counts as time in the air.

Total_time in the air is pretty objective: just a total arm time.
But idk, if you do not agree I can rever this file no problems.